lib/utils.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
does `goodparts` _enforce_ having the `'use strict';` at the top of our files...?